modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -2835,7 +2842,9 @@ def compute_groupby(df, drop=False, partition_idx=0):
                     result = pandas.DataFrame(index=grouped_df.size().index)
                 if isinstance(result, pandas.Series):
                     result = result.to_frame(
-                        result.name if result.name is not None else "__reduced__"
+                        result.name
+                        if result.name is not None
+                        else MODIN_UNNAMED_SERIES_LABEL
```

Comment:
```suggestion
                        result.name or MODIN_UNNAMED_SERIES_LABEL
```
?

Reply:
as above, this is incorrect when `result.name == ""`